templates/zsh.txt

```diff
@@ -59,9 +59,7 @@ function __zoxide_z() {
             echo "zoxide: \\$OLDPWD is not set"
             return 1
         fi
-    elif [ "$#" -eq 1 ] &&  cd "$1" &>{{ Opts::DEVNULL }}; then
-        __zoxide_cd "$1"
-    else
+    elif ! { [ "$#" -eq 1 ] && __zoxide_cd "$1" &>{{ Opts::DEVNULL }}; }; then
```

Comment:
This has a problem - it hides the output of `__zoxide_cd` as well as any other PWD hook that the user might have.